keyboards/fleuron/fleuron.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2017 REPLACE_WITH_YOUR_NAME
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef FLEURON_H
+#define FLEURON_H
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The following is an example using the Planck MIT layout
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
```

Comment:
Could you rename this to `LAYOUT_ortho_6x16`

Reply:
Would "LAYOUT_fleuron_grid" be ok? I'm planning on adding the following layouts that are supported by the pcb:
LAYOUT_fleuron_numpadright
LAYOUT_fleuron_numpadleft
LAYOUT_fleuron_numpadcenter
LAYOUT_fleuron_fatthumbnumpadright

Reply:
Ideally, the ortho one should be there, but you can add either:

```
#define LAYOUT_fleuron_grid LAYOUT_ortho_6x12
```
or
```
#define LAYOUT_ortho_6x12 LAYOUT_fleuron_grid
```

That way, both are supported.

Reply:
Done :)